service-registry/src/main/java/org/apache/servicecomb/serviceregistry/consumer/MicroserviceVersions.java

```diff
@@ -210,9 +220,9 @@ public void onMicroserviceInstanceChanged(MicroserviceInstanceChangedEvent chang
     //   if pull 1/2/3, and then delete 3, but "delete 3" received before pull result, will have wrong 3.
     // EXPIRE::
     //   black/white config in SC changed, we must refresh all data from sc.
-    postPullInstanceEvent(TimeUnit.MILLISECONDS.toMillis(1));
+    postPullInstanceEvent(0);
```

Comment:
Why do you change the time to zero.

Reply:
This is an event fired by SC of instance change and we need to get the changed instance immediately. Some user applications based on this event to make calls, and if we pending to get instance, will cause their logic more complex

Reply:
even time is 0, it's a async task.
everything not changed, except time gap is smaller, but time gap is not 0.

Reply:
but it's better to use 0 than 1, right?

Reply:
ok